adafruit_max31856.py

```diff
@@ -289,13 +300,15 @@ def reference_temperature_thresholds(self):  # pylint: disable=invalid-name
         )
 
     @reference_temperature_thresholds.setter
-    def reference_temperature_thresholds(self, val):  # pylint: disable=invalid-name
+    def reference_temperature_thresholds(  # pylint: disable=invalid-name,
+        self, val: Tuple[float, float]
+    ) -> None:  # pylint: disable=invalid-name
```

Comment:
Same thing here, I think it's this second `# pylint: disable=invalid-name` you can get rid of.